environment.yaml

```diff
@@ -10,7 +10,7 @@ dependencies:
   - click>=7.1.0
   - lxml
   - pip
-  - pystac
+  - pystac>=1.8.0
```

Comment:
please also add the new version requirement to `setup.py`.

Reply:
Done https://github.com/SAR-ARD/S1_NRB/pull/117/commits/1679d64b868c65a3b57c9b13d2730931360c6c11